src/WorkspaceContext.ts

```diff
@@ -372,7 +372,7 @@ export class WorkspaceContext implements vscode.Disposable {
         // find context with root folder
         const index = this.folders.findIndex(context => context.folder.fsPath === folder.fsPath);
         if (index !== -1) {
-            console.error(`Adding package folder ${folder} twice`);
+            console.warn(`Adding package folder ${folder} twice`);
```

Comment:
If this is going to change then I would just completely change it to use `outputChannel.log("Adding package folder twice", "WARN")` instead so that it appears in the logs.

Reply:
Good call, done